graphics/argyll/Portfile

```
@@ -64,7 +64,7 @@ post-destroot {
     # Install documentation.
     xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}
     xinstall -m 644 -W ${worksrcpath} \
-        notes.txt Readme.txt ttbd.txt \
+         Readme.txt \
```

Indentation is not aligned properly here.
```suggestion
        Readme.txt \
```